lib/ipa_test_kit/version.rb

```diff
@@ -1,3 +1,3 @@
 module IpaTestKit
-  VERSION = '0.1.0'
```

Comment:
Test kits / suites are versioned independently of IGs, because test development may occur separately.  If all you are doing is updating the IG package for resource-level validation, and not updating anything else, we could just leave it as-is, or set this to `VERSION = '0.1.1'` to reflect that not much has changed.